src/app/(portal)/[lng]/admin/dashboard/[projectCode]/[tab]/overview/partials/UsageGraph.tsx

```diff
@@ -55,7 +55,9 @@ const valueToLabelMap = {
 
 export default function UsageGraph() {
   const { dateRange: range } = useDashboardFilters()
-  const projectId = useProjectCode() ?? 0
+  const projectCode = useProjectCode()
+  const projectId = typeof projectCode === "number" ? projectCode : 0
+
   const { data: statsData, isLoading } = useOverviewStatsData(projectId, {
     range
   })
```

Comment:
can add 

```suggestion
const { data: statsData, isLoading } = useOverviewStatsData(projectId, {
  range,
  enabled: typeof projectCode === "number"  // Only fetch when projectCode is a number
})
```
 so that unnecessary api calls dont happen when the projectId fallsback to 0